### ISSUE TITLE
Specifying size for WES samples in count-reads 

### ISSUE THREAD

**Issue by bjlerman** [raphael-group/hatchet#199]
Hi, I see in the instructions for the WES demo that 250 kb should be selected as the bin size in count-reads, but I don't see that as an accepted argument. When I include --size 250kb in the code anyways it throws an error. The demo references -b as the argument for this but this is currently used to specify the .1bed file...is the size instruction from an older version of hatchet? If so, are there new recommendations for how to optimize for WES data?

Thanks so much!

**Comment by mmyers1** [comment]
You are correct, the WES demo is currently out-of-date. You can follow the instructions in the complete demo: https://raphael-group.github.io/hatchet/examples/demo-complete/demo-complete.html

The variable-width binning in recent HATCHet versions should adjust automatically to the coverage density in WES, but you may want to tune the minimum SNP-covering reads `msr` and minimum total reads `mtr` per bin if you find that the signal looks noisy. These are parameters to `combine_counts` step in HATCHet that implicitly modify the bin size: higher values produce larger bins on average, and vice versa.
